x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -314,6 +314,8 @@ const FormattedFieldValueComponent: React.FC<{
       value,
     });
   } else if (isUnifiedDataTable || columnNamesNotDraggable.includes(fieldName) || !isDraggable) {
```

Comment:
@logeekal I can see that if this renderer is used in unified data table, then we are not applying a specific renderer for the empty value